### ISSUE TITLE
Observed Crash when Scrolling the WebView

### ISSUE THREAD

**Issue by marchani** [gree/unity-webview#851]
I am seeing an intermittent crash with a modified version of the sample project on an Android phone when scrolling through the webpage displayed in a WebView in the following environment:

Unity Version: 2021.3.12f1
Android Version: 12
Phone: Google Pixel 5

I modified the sample project in the following way to reproduce the issue:

1. I created a new scene called Initial that contains two buttons: 'WebView 1' and 'WebView 2'.

2. When the 'WebView 1' button is clicked, it loads the existing Sample scene with the following URL: https://www.atlassian.com/legal/privacy-policy.

3. When the 'WebView 2' button is clicked, it loads the existing Sample scene with the following URL: https://www.youtube.com/playlist?list=PL1wNGMba9T63zX4ZtN4R7sxPdH7MQeob-.

4. I added a back button to the Sample scene to navigate back to the Initial scene that I created in Step 1.

Here are the steps to reproduce the crash that I am observing:

1. Click on the 'WebView 1' button
2. Scroll to the bottom of the webpage
3. Click the back button
4. Click on the 'WebView 2' button
5. Scroll down on the webpage
6. Click the back button
7. Repeat Steps #1 through #6 until the crash is observed

I am able to intermittently reproduce the crash on either Step 2 or Step 5.  However, it can be tedious to reproduce this issue since it does not occur often.  If you attempt to reproduce this issue, I recommend redeploying the Unity app to the mobile device, if the issue is not observed in the first few attempts to perform the steps to reproduce.

Please let me know if I can provide any additional information to help troubleshoot or resolve this issue.

Thanks in advance for your help.

**Comment by KojiNakamaru** [comment]
I modifed the sample app by following your note:

https://github.com/gree/unity-webview/tree/example-for-851

and I cannot reproduce crashes at least on my devices (Vankyo S7 (Android 9), OSCAL Pad 8 (Android 11)). Does this sample app reproduce the issue with your device? If it doesn't and your test code does, can you identify any difference between them?

Also, can you provide crash logs by `adb logcat`? Those might help to seek any clue.

**Comment by marchani** [comment]
Yes, I checked out the 'example-for-851' branch, built and ran the sample project, and I am able to reproduce the issue on my device (Google Pixel 5 with Android 12): intermittent crash when scrolling the WebView.

Attached are three logs that I captured after three different intermittent crashes by following the steps to reproduce using `adb logcat`.  In each of the three crashes, I stopped the logging a few seconds after the crash occurred.

Please let me know if I can provide any additional information to help narrow down this issue or if there are any steps that you would like me to test.

[logcat (reproduction 01).txt](https://github.com/gree/unity-webview/files/9984186/logcat.reproduction.01.txt)
[logcat (reproduction 02).txt](https://github.com/gree/unity-webview/files/9984187/logcat.reproduction.02.txt)
[logcat (reproduction 03).txt](https://github.com/gree/unity-webview/files/9984188/logcat.reproduction.03.txt)

**Comment by KojiNakamaru** [comment]
Thank you for the logs. Though 

https://stackoverflow.com/questions/66361472/app-crashed-with-base-apklibmonochrome-so

discusses about different devices/android versions, it and the linked page 

https://issuetracker.google.com/issues/242460140

could be helpful.
